### DISCUSSION TITLE
p/bad908e3481c4981a4da32a44275c4b4/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#52] [Announcements]
# p/bad908e3481c4981a4da32a44275c4b4/

五一假期一直在广州的朋友家中休息，中间也出去见了几个朋友，但并未真的出去玩。正是因为上次清明节在泰山的惨痛经历，让吾辈明白要避开节假日人流的道理。  桂林五一假期刚刚结束，吾辈就前往桂林。早就听闻桂林山水甲天下，来到之后风景确实还不错，但也遇到一些糟心的事，暂且按下不表。 桂林是一座绿化程度很高的城市，一些街拍感觉还不错。  但将镜头拉近，可以看到一些建筑已是残旧不堪，本应在墙体内的红砖直接裸露在

https://blog.rxliuli.com/p/bad908e3481c4981a4da32a44275c4b4/

**Comment by xupt-chenxi** [comment]
吾辈这是结束日本的留学，回国了？

**Reply by rxliuli** [reply]
嗯。
